README.md

```diff
@@ -205,6 +188,25 @@ by the `FLASK_CONFIG` environment variable.
     URL passed to NHS OIDC when the user is in the login flow. It should
     have the form
     `http(s)://<externally_reachable_app_url>/nhs-login-callback`.
+    
+  - `NHS_OIDC_LOGIN_PRIVATE_KEY_PATH`: This is used to specify the path 
+     to the private key pem file locally.
+     
+  - `NHS_OIDC_LOGIN_PRIVATE_KEY`: This is used to specify the private key
+     contents in the GOV PAAS environment only. 
+
+  - `AWS_RDS_DATABASE_NAME` **\[required\]**: This variable sets the
+    database that the app will use to persist form answers.
+
+  - `AWS_REGION` **\[required\]**: This variable sets the AWS region
+    that the app will communicate with to persist form answers.
```

Comment:
I think this (and instances below) will need to be `AWS_DEFAULT_REGION` - happy to be wrong though

Reply:
Looking at the code that value is passed into the boto3 client as a param so i think it should be okay .

![image](https://user-images.githubusercontent.com/5682400/91177570-d260b180-e6db-11ea-9e7a-3d5bf701df92.png)

Reply:
@danwilliams

Reply:
@thomasridd

Reply:
cool - if you're passing the region explicitly you can call it anything you fancy 👍